src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -2364,11 +2364,21 @@ internal sealed partial class LazyForLoopStatement : BaseForLoopStatement, IForL
     /// </summary>
     internal abstract partial class BaseForToLoopStatement : LoopStatement, IForToLoopOperation
```

Comment:
>BaseForToLoopStatement [](start = 36, length = 22)

Which change is causing this generated file to get updated? Was expecting to see a change to on  of our XML files or generators but not seeing that.

Reply:
@jaredpar We have still not hooked up the generator to this file: https://github.com/dotnet/roslyn/issues/20204. We should probably rename this file to remove the `.generated.cs` suffix until that is implemented.

Reply:
Ah okay. Thanks for the explanation.

As for renaming. I'm ambivalent. I don't want to create unnecessary work here if there is an issue tracking it. 

---
In reply to: [187120137](https://github.com/dotnet/roslyn/pull/26734#discussion_r187120137) [](ancestors = 187120137)

Reply:
I'm planning on addressing the issue during the hackathon.

---
In reply to: [187121166](https://github.com/dotnet/roslyn/pull/26734#discussion_r187121166) [](ancestors = 187121166,187120137)